pytorch_lightning/trainer/trainer.py

```diff
@@ -666,7 +673,7 @@ def run_test(self):
         # only load test dataloader for testing
         # self.reset_test_dataloader(ref_model)
         with self.profiler.profile("run_test_evaluation"):
-            eval_loop_results, _ = self.run_evaluation(test_mode=True)
+            eval_loop_results, eval_results = self.run_evaluation(test_mode=True)
```

Comment:
if `eval_results` is unused, can you keep the `_`? it makes the IDE not complain about unused variable